saleor/graphql/payment/tests/queries/test_query_payment.py

```diff
@@ -0,0 +1,74 @@
+import graphene
```

Comment:
Could we only change the name of this file to `test_payment_query.py` to keep the pattern, as we have already `test_payments_query.py`?

Reply:
I think we could put all queries to one file - there are only two queries.